streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -157,6 +157,7 @@
                                         ConsumerConfig.VALUE_DESERIALIZER_CLASS_DOC)
                                 .define(TIMESTAMP_EXTRACTOR_CLASS_CONFIG,
                                         Type.CLASS,
+                                        WALLCLOCK_TIMESTAMP_EXTRACTOR,
```

Comment:
@ymatsuda I thought it was actually merged in a previous PR for improving APIs, but obviously it gets left and slipped until now.

Let's discuss about this, I think your hesitation is that it should better to be a use's conscious decision which timestamp to use. Personally I would rather prefer to not enforce every user to deliberate on timestamps: generally speaking, I want to minimize the number of conscious decisions, i.e. required configs for users who use KStream for the first time, and ideally bootstrap brokers is the only thing you need to specify for your first KStream program. And as users start to use more they will dive into the details.

Thoughts?